public/app/features/plugins/sandbox/sandbox_components.tsx

```diff
@@ -70,17 +70,19 @@ export async function sandboxPluginComponents(
 
   // config pages
   if (Reflect.has(pluginObject, 'configPages')) {
-    const configPages: Record<string, PluginConfigPage<any>> = Reflect.get(pluginObject, 'configPages');
-    for (const [key, value] of Object.entries(configPages)) {
-      if (!value.body || !isFunction(value.body)) {
-        continue;
+    const configPages = Reflect.get(pluginObject, 'configPages');
```

Comment:
I added these changes myself and I can confirm they work as expected in regards to sandbox code.